packages/next/next-server/server/next-server.ts

```diff
@@ -59,9 +59,20 @@ import {
   setSprCache,
 } from './spr-cache'
 import { isBlockedPage } from './utils'
+import Worker from 'jest-worker'
 
 const getCustomRouteMatcher = pathMatch(true)
 
+const staticPathsWorker = new Worker(require.resolve('./static-paths-worker'), {
```

Comment:
This should be added to `next-dev-server` right?

Reply:
Right, updated it to be initialized in `next-dev-server` 👍